.github/workflows/test.yml

```diff
@@ -45,16 +45,29 @@ jobs:
       - name: Test example tvOS
         run: scripts/test.sh example-cross-platform tvos
 
-  upcoming_feature:
-    name: Upcoming features
+  test_upcoming_features:
+    name: Test upcoming features with Xcode 15
     runs-on: macos-14
     env:
       DEVELOPER_DIR: /Applications/Xcode_15.4.app
     steps:
       - uses: actions/checkout@v4
-      - name: Test with upcoming features
+      - name: Test upcoming features
         run: ENABLE_UPCOMING_FEATURES=1 scripts/test.sh library ios
 
+  test_language_mode:
+    name: Test Swift 5 language mode
+    runs-on: macos-14
+    strategy:
+      matrix:
```

Comment:
No xcode_version matrix? (compiler) I dont know whats default DEVELOPER_DIR but this means not both `compiler(5)` and `compiler(6)` are evaluated

Reply:
ah, all `compiler(<=6)` was changed to `swift`, didnt notice

Reply:
Default DEVELOPER_DIR is explicitly Xcode 16.
https://github.com/ra1028/swiftui-atom-properties/blob/9e5eb27d54d7f91985efd6a16e32a64fce17ff00/.github/workflows/test.yml#L13

Reply:
Then I guess `test_upcoming_features` does it with `Xcode_15.4` and `test_language_mode` uses default one (16+?)

Reply:
Yes

Xcode 16:Swift6 = `test`
Xcode 16:Swift6:Upcoming features = N/A
Xcode 16:Swift5 = `test_language_mode`
Xcode 16:Swift5:Upcoming features = `test_language_mode`
Xcode 15:Swift5 = `test`
Xcode 15:Swift5:Upcoming features = `test_upcoming_features`